apix-impl/src/test/java/eu/xenit/apix/tests/search/SearchFacetServiceUnitTest.java

```diff
@@ -168,15 +168,16 @@ public List<FacetSearchResult> initExpectedResult_for_assertThat_getFacetResults
         modifiedValues.add(modifiedFacetValue);
         modifiedResult.setValues(modifiedValues);
         expectedResult.add(modifiedResult);
-        FacetSearchResult createdResult = new FacetSearchResult();
-        createdResult.setName("{http://www.alfresco.org/model/content/1.0}created");
-        List<FacetValue> createdValues = new ArrayList<>();
-        FacetValue createdFacetValue = new FacetValue();
-        createdFacetValue.setValue("[2020-08-31T07:00:00.000Z TO 2023-09-02T10:01:00.000Z]");
-        createdFacetValue.setCount(1);
-        createdValues.add(createdFacetValue);
-        createdResult.setValues(createdValues);
-        expectedResult.add(createdResult);
+// ALFREDAPI-544 or ALFREDAPI-549 remove facetresult in case of pure datetime range
```

Comment:
Can we move this test to the Search Service and validate the returned properties and facets?